crates/bin/pindexer/src/governance/schema.sql

```diff
@@ -0,0 +1,75 @@
+CREATE TYPE proposal_stage AS ENUM ('VOTING', 'FINISHED');
+CREATE TYPE proposal_status AS ENUM ('VOTING', 'WITHDRAWN', 'FINISHED', 'CLAIMED');
+CREATE TYPE payload_type AS ENUM ('SIGNALING', 'EMERGENCY', 'PARAMETER_CHANGE', 'COMMUNITY_POOL_SPEND', 'UPGRADE_PLAN', 'FREEZE_IBC_CLIENT', 'UNFREEZE_IBC_CLIENT');
+CREATE TYPE vote_type AS ENUM ('ABSTAIN', 'YES', 'NO');
+CREATE TYPE proposal_outcome AS ENUM ('PASSED', 'FAILED', 'SLASHED');
+
+CREATE TABLE governance_proposals (
```

Comment:
This table needs to have a name that prefixes the module path, like the other tables, so that it's more composable with other `AppView` implementations.